utils.py

```diff
@@ -5,6 +5,8 @@
 import click
 import mne
 from mne.utils import logger
+from pandas import read_csv
```

Comment:
I would do "import pandas as pd" and use pd.read\_csv --> it's very likely that we'll need other pandas funcs in this script soon